src/fundamental/macro-fundamental.h

```diff
@@ -105,8 +105,8 @@
                 _expr_;                                  \
         })
 
-#define assert_cc(expr) static_assert(expr, #expr)
-
+#define assert_message_cc(expr, message) static_assert(expr, message)
```

Comment:
That's literally what static_assert is for.

Reply:
yes obviously